### ISSUE TITLE
js class names

### ISSUE THREAD

**Issue by iamjoshellis** [wearelighthouse/lightbones-scss#1]
I think there should be a denomination assigned to 'stateful' classes that are added via js. e.g. `.js--state__success`.

**Comment by cjquinn** [comment]
Yeah I'm big on this, my thoughts were that we would use camel case where ever classes/ids were being used as hooks in JavaScript.

**Comment by cjquinn** [comment]
And that no css should be applied to a class that is used in JavaScript.
